tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py37, py38
+
+[testenv]
+deps = 
+	-r requirements_test.txt
+	.[deep-learning]
+
+commands = python -m spacy download en_core_web_sm
+		   pytest -m  "not (integration)" -s -v --durations=0 --disable-warnings --tb=line --cov=wellcomeml ./tests
```

Comment:
is there an indentation problem here?

Reply:
ini files do not care about indentation so I guess that is why this works but let me see whether this is a space vs tabs issue

Reply:
7888bd292e74f569703ec428a5485c87b1ca4860